examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads 
+--------------------
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a geopandas.GeoDataFrame object.
```

Comment:
Include link to https://geopandas.org/docs/reference/api/geopandas.GeoDataFrame.html#geopandas.GeoDataFrame.
```suggestion
Roads
-----
The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
stored in a :class:`geopandas.GeoDataFrame` object.
```